packages/messaging/ios/RCTAEPMessaging.xcodeproj/project.pbxproj

```diff
@@ -0,0 +1,306 @@
+// !$*UTF8*$!
+{
+	archiveVersion = 1;
+	classes = {
+	};
+	objectVersion = 46;
+	objects = {
+
+/* Begin PBXBuildFile section */
+		09037ACA74711BE35A9507C5 /* libPods-RCTACPUserProfile.a in Frameworks */ = {isa = PBXBuildFile; fileRef = 13DF1315553E7A765C495C83 /* libPods-RCTACPUserProfile.a */; };
```

Comment:
If we don't use userprofile for for Messaging, then change the userprofile to messaging under Frameworks.